packages/paste-core/components/spinner/src/index.tsx

```diff
@@ -14,7 +14,10 @@ export interface SpinnerProps extends IconWrapperProps {
 }
 
 export const Spinner = React.forwardRef<HTMLDivElement, SpinnerProps>(
-  ({size, color = 'currentColor', title, as, display, decorative, delay = 250}, ref) => {
+  (
+    {size, color = 'currentColor', title, as, display, decorative, delay = 250, element = 'SPINNER_ICON_WRAPPER'},
```

Comment:
Suggest matching component name
```suggestion
    {size, color = 'currentColor', title, as, display, decorative, delay = 250, element = 'SPINNER'},
```
The rest of this file is fine, just this one change